src/gen_bpf.c

```diff
@@ -1291,6 +1291,50 @@ static int _gen_bpf_insert(struct bpf_state *state, struct bpf_instr *instr,
 	return rc;
 }
 
+/**
+ * Omit syscall from the BPF filter?
```

Comment:
Big time nitpick, but typically the one-liner summaries are statements not questions, e.g. "Decide if we need to omit the syscall from the BPF filter".

Reply:
:) Can you tell I was in a Java frame of mind?  Thanks, I like your wording better.